apps/qubit/modules/search/templates/_box.php

```diff
@@ -8,9 +8,9 @@
     <input type="hidden" name="sort" value="relevance"/>
 
     <?php if (isset($repository) && !sfConfig::get('app_enable_institutional_scoping')): ?>
-      <input type="text" name="query"<?php if (isset($sf_request->query)) echo ' class="focused"' ?> value="<?php echo $sf_request->query ?>" placeholder="<?php echo __('Search %1%', array('%1%' => strip_markdown($repository))) ?>"/>
```

Comment:
Looks good!  I think this change needs to also be made in two additional places:
- In the equivalent file in the Archives Canada plugin
- ~line 30 in: apps/qubit/modules/repository/templates/_holdingsInstitution.php

Reply:
Thanks Steve! Will do.

Reply:
(I've committed escaping to those other templates.)